elements/nuxeo-app/nuxeo-expired-session.js

```diff
@@ -36,11 +36,10 @@ Polymer({
         height: var(--nuxeo-drawer-header-height);
         padding-left: 16px;
         position: fixed;
-        top: 0;
+        top: var(--nuxeo-app-top);
         left: 0;
         right: 0;
         z-index: 1000;
-        visibility: hidden;
         transform: translateY(-100%);
         -webkit-transform: translateY(-100%);
```

Comment:
Can you please consider removing this obsolete line? 🙏 
```suggestion
        -webkit-transform: translateY(-100%);
```

Reply:
done